tests/beman/execution/CMakeLists.txt

```diff
@@ -105,51 +98,9 @@ if(NOT DEFINED CMAKE_CXX_STANDARD)
   set(CMAKE_CXX_STANDARD 23)
 endif()
 
-if(NOT DEFINED CMAKE_PREFIX_PATH)
-  set(CMAKE_PREFIX_PATH ${CMAKE_INSTALL_PREFIX})
-endif()
-
-if(PROJECT_IS_TOP_LEVEL)
-  find_package(beman_execution 0.0.1 EXACT QUIET)
-  if(beman_execution_FOUND)
```

Comment:
... den sie wissen nicht was sie tun!

Reply:
I do understand the German but I don't quite understand the implication. I admit that I don't understand what that is trying to achieve. There is certainly no equivalent logic in the `exemplar` version of the file. Explicitly testing for a version number seems ill-advised, though.